cmplibrary/gradle.properties

```diff
@@ -1,4 +1,4 @@
-VERSION_NAME = 7.6.1
+VERSION_NAME = dia-2024
```

Comment:
before merging we can revert this change

Reply:
yup, done